docs/credits.rst

```diff
@@ -227,7 +227,7 @@ Other Credits
 * Andrew Pontzen and the `pynbody <https://github.com/pynbody/pynbody>`_ team
   (For code that grew into :mod:`astropy.units`)
 * Everyone on `astropy-dev <http://groups.google.com/group/astropy-dev>`_
-  and the `astropy mailing list <http://mail.scipy.org/mailman/listinfo/astropy>`_
+  and the `astropy mailing list <https://mail.python.org/mailman/listinfo/astropy>`_
```

Comment:
Sphinx complains about duplicate target name https://travis-ci.org/astropy/astropy/jobs/217620466#L3377. Could you please use the same url (http --> https) everywhere (e.g. in docs/index.rst)? I think that should solve the issue.